src/Controller/ReservationController.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Reservation;
+use App\Form\ReservationType;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+class ReservationController extends AbstractController
+{
+    /**
+     * @Route("/reservation", name="reservation")
+     */
+    public function index(Request $request)
+    {
+        $dateFrom = new \DateTime('2018-11-27');
+        $sectorName = "Trečias Sektorius";
+        $amount = 30;
+        $house = 0;
+        $userId = 1;
+        $hours = 36;
+
+        if ($sectorName === "Trečias Sektorius") {
+            $house = 1;
+        }
+
+        $reservation = new Reservation();
+        $form = $this->createForm(ReservationType::class, $reservation);
+
+        $form->handleRequest($request);
+        if ($form->isSubmitted() && $form->isValid()) {
+            $timeFrom = $request->request->get('timeFrom');
```

Comment:
kam imami `GET` parametrai, jei submitinama forma? https://stackoverflow.com/questions/28578191/symfony2-saving-form-data-in-a-single-step
tikrai tarp mano nurodytu video buvo parodyta kaip tinkamai submitinti forma

Reply:
pabandyk https://symfony.com/blog/new-and-improved-generators-for-makerbundle tiesiog cruda rezervaciju enticiui padaryt. ir gales perkilnot dali `actionu` i admin skyriu, kaip: edit, delete, net ir new